### PR TITLE
Codes for A5

### PR DESCRIPTION
**Pull request by cpcplus**
Codes for A5

### CODE REVIEW

**Comment by bradymiller** [comment]
hi @cpcplus ,
Congrats on first github PR!
On your local repository, recommend moving the database.sql file to sql/database.sql, committing it, and then pushing the changes in your master branch to your github repo. That will then also automatically fix this PR (so we will then be able to easily see your work in the sql/database.sql file).

**Comment by tywrenn** [comment]
@bradymiller I'll mark where he inserted code

File: database.sql [hunk 1]
```diff
@@ -0,0 +1,10616 @@
...
+  `dlm_provider` int(10) unsigned DEFAULT NULL,
+  `dlm_sign_height` double NOT NULL,
+  `dlm_sign_width` double NOT NULL,
+  `dlm_filename` varchar(45) NOT NULL,
+  `dlm_effective_date` datetime NOT NULL,
+  `dlm_version` int(10) unsigned NOT NULL,
+  `content` varchar(255) NOT NULL,
+  `dlm_savedsign` varchar(255) DEFAULT NULL COMMENT '0-Yes 1-No',
+  `dlm_review` varchar(255) DEFAULT NULL COMMENT '0-Yes 1-No',
+  `dlm_upload_type` tinyint(4) DEFAULT '0' COMMENT '0-Provide
```
Comment (tywrenn) [inline database.sql]: Code insertion here

Comment (tywrenn) [inline database.sql]: Code insertion here

Comment (tywrenn) [inline database.sql]: Code Removal after line 4887:
``sql_version` VARCHAR(150) NOT NULL,
  `acl_version` VARCHAR(150) NOT NULL,`
Removed lines 4888 and 4889

**Comment by cpcplus** [comment]
Hi guys,
I ran 'docker-compose up' after making the following changes:
1) Removing/commenting out these lines:
    ALTER TABLE `patient_data` ADD `sexual_orientation` varchar(255) DEFAULT NULL;
    ALTER TABLE `patient_data` ADD `gender_identity` varchar(255) DEFAULT NULL;
2) Add columns sexual_orientation and gender_identity directly into the CREATE TABLE patient_data statement itself.

Looking fine after running 'docker-compose up'

**Comment by cpcplus** [comment]
@bradymiller, @tywrenn 
Hi guys,
I ran 'docker-compose up' after making the following changes:

    Removing/commenting out these lines:
    ALTER TABLE patient_data ADD sexual_orientation varchar(255) DEFAULT NULL;
    ALTER TABLE patient_data ADD gender_identity varchar(255) DEFAULT NULL;
    Add columns sexual_orientation and gender_identity directly into the CREATE TABLE patient_data statement itself.

Looking fine after running 'docker-compose up' @

**Pull request closed without merging**